### PR TITLE
Update to fetch study

### PR DESCRIPTION
**Pull request by mortonjt**
Saves samples fetched from EBI to separate folders

### CODE REVIEW

File: ipynb/primary-processing/1-get_sequences_and_metadata.md [hunk 1]
```diff
@@ -29,17 +29,10 @@ Now let's actually fetch the study data. `fetch_study` will only pull down acces
...
->>> !cat $bash_compatible_sequence_files > $agp_sequences
-...
->>> if len(accessions) > 1:
-...     qiime_compatible_mapping_files = ','.join([form_path(acc, 'txt') for acc in accessions])
-...     !merge_mapping_files.py -m $qiime_compatible_mapping_files -o $agp_metadata
->>> else:
-...     src = form_path(accessions[0], 'txt')
-...     !cp $src $agp_metadata
+>>> sample_sequence_files = ' '.join(agenv.get_files(chp_path, suffix='fna'))
+>>> !cat $sample_sequence_files > $agp_sequences
```
Comment (wasade) [inline ipynb/primary-processing/1-get_sequences_and_metadata.md]: it's likely that this won't work when there are thousands of files due to the bash command line length limit

**Comment by wasade** [comment]
it looks like it isn't correctly recognizing that the test data are present?

File: americangut/notebook_environment.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 from americangut.util import get_existing_path
 
 
-_TEST_ENV = os.environ.get('AG_TESTING') == 'True'
-_EBI_ACCESSIONS = ['ERP012511']
+_EBI_ACCESSIONS = ['ERP012803']
```
Comment (squirrelo) [inline americangut/notebook_environment.py]: Hooray umbrella project!

File: americangut/notebook_environment.py [hunk 3]
```diff
@@ -352,13 +347,41 @@ def get_accessions():
...
 
 
+def get_files(rootdir, suffix):
+    """Get the filepaths with a given suffix
+
+    Parameters
+    ----------
+    rootdir : str
+        The root directory to look under
+
```
Comment (squirrelo) [inline americangut/notebook_environment.py]: need to add suffix parameter

File: ipynb/primary-processing/01-get_sequences_and_metadata.md [hunk 4]
```diff
@@ -16,31 +16,25 @@ First, let's setup and sanity check our environment.
...
 ```
 
 Now that we have the sequences and sample information, let's merge all the data into a single file to ease downstream processing.
 
 ```python
->>> chp_base = os.path.split(chp_path)[-1]
->>> form_path = lambda acc, ext: agenv.get_existing_path(os.path.join(chp_base, '%s.%s' % (acc, ext)))
+>>> sample_sequence_files = agenv.get_files(chp_path, suffix='fna')
+>>> for f in sample_sequence_files:
+...     !cat $f >> $agp_sequences
```
Comment (mortonjt) [inline ipynb/primary-processing/01-get_sequences_and_metadata.md]: Good call on this one

**Comment by mortonjt** [comment]
Ready to merge?

**Comment by wasade** [comment]
am all :+1: for it :)

**Pull request merged**